src/util/operations.spec.tsx

```diff
@@ -44,3 +44,21 @@ describe("getInstanceName", () => {
     expect(name).toBe("testInstance3");
   });
 });
+
+describe("getProjectName", () => {
+  it("identifies project name from an instance operation when no project parameter is present", () => {
+    const operation = craftOperation("/1.0/instances/testInstance1");
+    const name = getProjectName(operation);
+
+    expect(name).toBe("default");
+  });
+
+  it("identifies project name from an instance operation in a custom project", () => {
+    const operation = craftOperation(
+      "/1.0/instances/testInstance2?project=project"
```

Comment:
Better use a different value, to ensure we don't accidentally return the key and accept it in the test, something like `project=fooProject`.